uspace/lib/c/test/main.c

```diff
@@ -40,19 +40,19 @@ PCUT_IMPORT(odict);
 PCUT_IMPORT(perf);
 PCUT_IMPORT(perm);
 PCUT_IMPORT(qsort);
+PCUT_IMPORT(ieee_double);
+PCUT_IMPORT(double_to_str);
 PCUT_IMPORT(scanf);
 PCUT_IMPORT(sprintf);
 PCUT_IMPORT(stdio);
 PCUT_IMPORT(stdlib);
 PCUT_IMPORT(str);
 PCUT_IMPORT(string);
 PCUT_IMPORT(table);
+PCUT_IMPORT(imath);
 PCUT_IMPORT(cap);
 PCUT_IMPORT(gsort);
-PCUT_IMPORT(ieee_double);
-PCUT_IMPORT(double_to_str);
 PCUT_IMPORT(getopt);
 PCUT_IMPORT(uuid);
-PCUT_IMPORT(imath);
```

Comment:
Could your commit message explain why the order was wrong, and why this change fixes it?